contribs/gmf/src/print/component.js

```diff
@@ -1283,25 +1291,55 @@ class Controller {
     return legend['classes'].length > 0 ? legend : null;
   }
 
+  /**
+   * @typedef {Object} LegendURLDPI
+   * @property {string} url The URL
+   * @property {number} dpi The DPI
+   */
 
   /**
    * Return the metadata legendImage of a layer from the found corresponding node
    * or undefined.
    * @param {string} layerName a layer name.
-   * @return {string|undefined} The legendImage or undefined.
+   * @param {number} [dpi=72] the image DPI.
+   * @return {LegendURLDPI|undefined} The legendImage with selected DPI or undefined.
    * @private
    */
-  getMetadataLegendImage_(layerName) {
+  getMetadataLegendImage_(layerName, dpi) {
```

Comment:
```suggestion
  getMetadataLegendImage_(layerName, dpi=72) {
```
And remove the next 3 lines ?